### ISSUE TITLE
don't exists some fieds in segment stats

### ISSUE THREAD

**Issue by Qavai** [bentasker/PHP-GPX-Ingest#12]
avgdeceleration
avgacceleration
...
maxacceleration

and so on

and recordedDuration

also dont exists fields elevation->min and max for all file

**Comment by bentasker** [comment]
Yes, this is noted in the documentation - those stats haven't been added yet at those levels. It's been at the back of my mind as something to get around to for a while, I've scheduled it under GPXIN-29 to make sure it gets implemented sooner rather than later
